Content.Server/Chemistry/Components/HyposprayComponent.cs

```diff
@@ -31,7 +32,8 @@ public sealed class HyposprayComponent : SharedHyposprayComponent, ISolutionChan
         [DataField("InjectSound")]
         private SoundSpecifier _injectSound = new SoundPathSpecifier("/Audio/Items/hypospray.ogg");
 
-        [ComponentDependency] private readonly SolutionContainerComponent? _solution = default!;
+        [DataField("solution")]
+        private string Solution { get; set; } = "hypospray";
```

Comment:
Would this string ever differ for a hypospray? Probably better to just make it a const string or just do like containers do and assume the component always has the same solution tbh.